buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/PropertiesHelper.kt

```diff
@@ -0,0 +1,22 @@
+package com.virtuslab.gitmachete.buildsrc
+
+import org.gradle.internal.util.PropertiesUtils
+import java.io.File
+import java.util.*
+
+object PropertiesHelper {
+  fun Properties.getPropertyOrNullIfEmpty(key: String): String? {
+    val value = getProperty(key)
+    return if (value == "") null else value
+  }
+
+  fun getProperties(file: File): Properties {
+    val properties = Properties()
+    properties.load(file.inputStream())
+    return properties
+  }
+
+  fun storeProperties(properties: Properties, file: File) {
+    PropertiesUtils.store(properties, file)
```

Comment:
This is taken from Gradle's own task `WriteProperties`; saves properties in abc order + without timestamp

Reply:
This function is a copy paste of the `PropertiesUtils.store`. Maybe we do not need this `fun storeProperties(properties: Properties, file: File)` and we can use `PropertiesUtils.store` directly in places where `storeProperties` is used? Looks like unnecsessary duplication to me 🤔

Reply:
Thought of the same... although still, I'd like to keep it under the helper to have somewhat uniform access to all property helper methods

Reply:
Esp. that the implementation might change one day (as it depends on internal Gradle stuff)

Reply:
Ok, makes sense 👍